Description: Disable Neon feature (requires unstable Rust on ARM64)
Forwarded: not-needed
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,6 @@ version = "0.2"
 optional = true
 
 [features]
-c_bindings = ["libc"]
 cmyk = []
 default = [
     "iccv4-enabled",
